content/_data/upgrades/2-221-1.adoc

```diff
@@ -0,0 +1,79 @@
+
+==== Removed Agent Protocols
+
+https://jenkins.io/security/advisory/2020-01-29/#SECURITY-1659[SECURITY-1659] and Remoting changelogs for https://github.com/jenkinsci/remoting/releases/tag/remoting-4.2[4.2],
+https://github.com/jenkinsci/remoting/releases/tag/remoting-4.0.1[4.0.1],
+https://github.com/jenkinsci/remoting/releases/tag/remoting-4.0[4.0],
+https://github.com/jenkinsci/remoting/releases/tag/remoting-3.40[3.40],
+https://github.com/jenkinsci/remoting/releases/tag/remoting-3.39[3.39].
+https://github.com/jenkinsci/remoting/releases/tag/remoting-3.38[3.38], and
+https://github.com/jenkinsci/remoting/releases/tag/remoting-3.37[3.37]
+
+The Remoting library has been updated to resolve a security issue in remoting versions prior to 4.0.1 and to add support for the <<websocket-protocol,WebSocket protocol>>.
+The update removes outdated agent protocols that had previously been link:/blog/2017/08/11/remoting-update/[deprecated in Jenkins 2.75].
+`Inbound TCP Agent Protocols` versions 1, 2, and 3 have been removed.
+
+<<websocket-protocol,WebSocket protocol>> connections require agents using Remoting 4.2 or later.
+
+plugin:ssh-slaves[SSH Build Agents] will always use the latest `agent.jar`, but other agent types may need to have their `agent.jar` updated manually.
+The following agent types should be updated:
+
+* Inbound TCP Agent Protocol agents (formerly known as Java Web Start agents) with `agent.jar` / `slave.jar` on the disk should update to version 4.2 or newer.
+  The latest `agent.jar` can be downloaded from any Jenkins server at `/jnlpJars/agent.jar` as in \https://your-jenkins-server.example.com/jnlpJars/agent.jar.
+  Users running Windows agents as a service with the Windows Service Wrapper should refer to https://github.com/jenkinsci/windows-slave-installer-module#upgrading-old-agents[Upgrading Old Agents] for information on agent upgrades
+* plugin:swarm[Swarm Client] should update to version 3.18 or newer
+* Agent Docker packages offered by the Jenkins project (https://hub.docker.com/r/jenkins/slave/[jenkins/slave] and https://hub.docker.com/r/jenkins/jnlp-slave/[jenkins/jnlp-slave]) should update to version https://github.com/jenkinsci/docker-slave/releases/tag/4.0.1-1/[4.0.1-1] or newer
+
+==== Security Defaults
+
+Some global security settings no longer have a user interface setting that will disable them.
+Previously, administrators could disable security with a single checkbox.
+They could also disable cross-site request forgery (CSRF) protection with a single checkbox.
+Those two settings **cannot be disabled** in this release.
+
+Password auto-fill is only allowed on the login password field.
+Password fields which are **not** the login password field will no longer allow password auto-fill.
+This avoids browser prompts to automatically fill password fields that are not the login password field.
+Password auto-fill for other password fields can be enabled with the system property:
```

Comment:
That's not what this is for. It's in case of unforeseen problems with the new password UI.